lib/Common/Memory/RecyclerTelemetryInfo.h

```diff
@@ -85,6 +85,28 @@ namespace Memory
 #endif
     };
 
+    /**
+     * Consolidated summary of data from RecyclerFlagsTable that we want to
+     * transmit via telemetry. Goal is to pack this into maximum of 64-bits.
+     */
+    union RecyclerFlagsTableSummary
+    {
+        struct fields_t
+        {
+            bool isMemProtectMode : 1;
+            bool isConcurrentEnabled : 1;
+            bool enableScanInteriorPointers : 1;
+            bool enableScanImplicitRoots : 1;
+            bool disableCollectOnAllocationHeuristics : 1;
+            bool recyclerStress : 1;
+            bool recyclerBackgroundStress : 1;
+            bool recyclerConcurrentStress : 1;
+            bool recyclerConcurrentRepeatStress : 1;
+            bool recyclerPartialStress : 1;
+        } fields;
+        uint32 bits;
```

Comment:
Same comment about using `bits` to access what was loaded into `fields` (technically undefined behavior), but this version is most likely going to work in a reasonable implementation of unions.

Reply:
Assuming the compiler for this code is always known (code not present in xplat builds for example), then implementation-defined/undefined behavior is a non-issue here. Sent some discussion offline.

Reply:
Thanks Doug, I'll add a comment to the code here that this may break on different compilers.